"4294967295" New messages

A few areas show "4294967295" new messages in the area list instead of "0". Like:

```
║ Area EchoID             Msgs        New║
║  42+ UTF-8                49 4294967295║
```
Gossiped 2.0.7 ARMv7 on Raspian Buster

I also can not enter that areas
```
$ gossiped
2020/01/16 09:17:43 gossipEd-linux/arm 2.0.7-b274c49a started
panic: runtime error: index out of range [49] with length 49 [recovered]
        panic: runtime error: index out of range [49] with length 49

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1(0xdb4be0)
        /home/travis/gopath/pkg/mod/github.com/rivo/tview@v0.0.0-20200108161608-1316ea7a4b35/application.go:149 +0x6c
panic(0x2064b0, 0x10fbb50)
        /home/travis/.gimme/versions/go1.13.linux.amd64/src/runtime/panic.go:679 +0x194
github.com/askovpen/gossiped/pkg/msgapi.(*JAM).GetMsg(0xd6a640, 0x32, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/askovpen/gossiped/pkg/msgapi/jam.go:107 +0x1588
github.com/askovpen/gossiped/pkg/ui.(*App).ViewMsg(0xd6acc0, 0x2a, 0x32, 0x0, 0x2, 0x10fbb20, 0x10, 0xd6ad30)
        /home/travis/gopath/src/github.com/askovpen/gossiped/pkg/ui/viewmsg.go:16 +0x50
github.com/askovpen/gossiped/pkg/ui.(*App).onSelected(0xd6acc0, 0x2b, 0x0)
        /home/travis/gopath/src/github.com/askovpen/gossiped/pkg/ui/arealist.go:105 +0x2c4
github.com/askovpen/gossiped/pkg/ui.(*App).AreaList.func2(0x2b, 0x0)
        /home/travis/gopath/src/github.com/askovpen/gossiped/pkg/ui/arealist.go:48 +0x2c
github.com/rivo/tview.(*Table).InputHandler.func1(0xcee8a0, 0xc0e490)
        /home/travis/gopath/pkg/mod/github.com/rivo/tview@v0.0.0-20200108161608-1316ea7a4b35/table.go:1163 +0x2b4
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xcee8a0, 0xc0e490)
        /home/travis/gopath/pkg/mod/github.com/rivo/tview@v0.0.0-20200108161608-1316ea7a4b35/box.go:160 +0x50
github.com/rivo/tview.(*Application).Run(0xdb4be0, 0x0, 0x0)
        /home/travis/gopath/pkg/mod/github.com/rivo/tview@v0.0.0-20200108161608-1316ea7a4b35/application.go:234 +0x2c8
github.com/askovpen/gossiped/pkg/ui.(*App).Run(0xd6acc0, 0x1, 0x1)
        /home/travis/gopath/src/github.com/askovpen/gossiped/pkg/ui/app.go:38 +0x44
main.main()
        /home/travis/gopath/src/github.com/askovpen/gossiped/gossiped.go:74 +0x3c0

```

--- a/pkg/msgapi/jam.go
+++ b/pkg/msgapi/jam.go
@@ -310,7 +310,9 @@ func (j *JAM) GetLast() uint32 {
 	j.readJLR()
 	for _, l := range j.lastRead {
 		if l.UserCRC == crc32r(config.Config.Username) {
-			// log.Printf("GetLast()->%d",j.getPositionOfJamMsg(l.LastReadMsg) + 1)
+			if j.getPositionOfJamMsg(l.LastReadMsg)+1 > uint32(len(j.indexStructure)) {
+				return uint32(len(j.indexStructure))
+			}
 			return j.getPositionOfJamMsg(l.LastReadMsg) + 1
 		}
 	}

